### ISSUE TITLE
automatically create mods and mods-unpacked directories

### ISSUE THREAD

**Issue by Qubus0** [GodotModding/godot-mod-loader#34]
for ease of setup for everyone

**Comment by ithinkandicode** [comment]
Yes to `mods`, no to `mods-unpacked`. The unpacked dir is an internal dir for modders creating mods in the editor, and shouldn't be exposed to general users. We don't want to confuse them or risk them putting files there that shouldn't be there

**Comment by otDan** [comment]
I guess the auto setup can be done by the dev tools for the mods-unpacked folder

**Comment by Qubus0** [comment]
that's a good idea. or we use a "dev-mode" switch

**Comment by ithinkandicode** [comment]
This was implemented in PR #81, [here](https://github.com/GodotModding/godot-mod-loader/pull/81/files/5b8f46cf92f0b04b4877546595712f44e0221100#diff-adbb9aa2d2a2827e3d03e1ef8548506f0cde9dd2679093b1926a33f3c2964fa5). I suggested it would be better in its own PR, so it may be partitioned off.

However, I don't know if we need to/should do this. It's helpful if you're adding mods manually, but if you're using Steam Workshop then your mods won't be in the EXE directory, and if you're using Thunderstore, then I think *(and I might be wrong)* they have mods stored separately, with a CLI arg to specify the mods dir. For those cases, having a "mods" folder just appear in your game folder will no doubt be confusing to users.

So I think we can probably close this. It's a great idea, but not really needed considering the full scope of ModLoader's use cases.
